Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.h

```diff
@@ -67,6 +67,26 @@ class WebChromeClient final : public WebCore::ChromeClient {
     void relayAccessibilityNotification(String&&, RetainPtr<NSData>&&) const final;
 #endif
 
+    bool isHardwareKeyboardAttached() const override { return m_isAttached; }
+
+    void startHardwareKeyboardAttachmentNotifications(HardwareKeyboardAttachmentCallback&& cb) override
```

Comment:
The style guide suggests going with full names, unless the abbreviation is canonical
```suggestion
    void startHardwareKeyboardAttachmentNotifications(HardwareKeyboardAttachmentCallback&& callback) override
```